### PR TITLE
feat: implement providers

### PR DESCRIPTION
**Pull request by MDavidson17**

### CODE REVIEW

File: topo_processor/metadata/metadata_loaders/tests/metadata_loader_imagery_historic_test.py [hunk 1]
```diff
@@ -308,3 +308,53 @@ def test_mission_metadata_not_added():
     metadata_loader_imagery_historic = MetadataLoaderImageryHistoric()
     title = metadata_loader_imagery_historic.get_title(metadata["survey"], metadata["alternate_survey_name"])
     assert title is None
+
+
+def test_test():
```
Comment (paulfouquet) [inline topo_processor/metadata/metadata_loaders/tests/metadata_loader_imagery_historic_test.py:313]: You haven't been inspired for the name of your test! Maybe something like `test_provider_added`?

File: topo_processor/stac/collection.py [hunk 2]
```diff
@@ -13,7 +13,14 @@
 
 from topo_processor.util import Validity
 
-GLOBAL_PROVIDERS = [pystac.Provider(name="LINZ", description="Land Information New Zealand", roles=["host"])]
+GLOBAL_PROVIDERS = [
```
Comment (paulfouquet) [inline topo_processor/stac/collection.py:16]: At some point if we need to use a same `provider` in different classes, that would be good to have a dedicated provider definition file.

**Review by paulfouquet — CHANGES_REQUESTED** [review]
Looks good, just the test name to be more specific.

File: topo_processor/metadata/metadata_loaders/tests/metadata_loader_imagery_historic_test.py (hunk 1 above)
Comment (MDavidson17) [reply topo_processor/metadata/metadata_loaders/tests/metadata_loader_imagery_historic_test.py:313]: :joy: oops!

**Review by blacha — APPROVED** [review]

**Review by amfage — APPROVED** [review]
:+1:

**Review by paulfouquet — APPROVED** [review]

**Pull request merged**